migrations/906-abuse-report-read.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE `abuse_reports`
+    ADD COLUMN `read` bool NOT NULL,
```

Comment:
The default for a new note would be false, but we want all the existing abuse reports to be marked read (so as to not have to read every existing report from the beginning of time).  Should I add DEFAULT true to the database to address the existing notes and leave the django model as default=false so they get created unread?